api/config/config_preprod.json

```diff
@@ -47,8 +47,8 @@
     "operationalTerraWorkspaceName": "aouwgscohortextractionpreprod",
     "operationalTerraWorkspaceBucket": "fc-5d8e6afe-9429-44ae-a22a-67b09705d1b4",
     "extractionPetServiceAccount": "pet-110393474898566940625@aouwgscohortextraction-preprod.iam.gserviceaccount.com",
-    "extractionMethodConfigurationNamespace": "aouwgscohortextraction-preprod",
-    "extractionMethodConfigurationName": "",
+    "extractionMethodConfigurationNamespace": "aouwgscohortextraction-preprod-manual",
```

Comment:
THanks, this is better - but why the "manual" suffix? Will there ever be a version that's not manual? And if so - when?

Reply:
manual implies that I uploaded the WDLs myself through the Agora UI. The main distinction being that my pmi-ops account is the owner of the "manual" method configuration namespace while the service account is the owner of the preset namespace. I was planning to run the Agora upload tool once the GATK PR merges which will use the SA owned namespaces. Although.. there is a little bit of work in figuring out how to import the WDL. I think I have a relatively low effort solution but it will still require some changes to the upload tool.